crates/hir_def/src/builtin_attr.rs

```diff
@@ -0,0 +1,428 @@
+//! Builtin attributes resolved by nameres.
+//!
+//! The actual definitions were copied from rustc's `compiler/rustc_feature/src/builtin_attrs.rs`.
```

Comment:
Can you add the exact commit you copied it from? It could be useful in the future.

Reply:
Yeah, good idea. Opened https://github.com/rust-analyzer/rust-analyzer/pull/6832.